### PR TITLE
Add Docker

### PR DESCRIPTION
**Pull request by inductor**
## Hey, I just made a Pull Request!

- Add Docker to try out

Just like https://github.com/spotify/backstage/pull/277 this PR improves Developer Experience only so tell me if I need to check all the stuff below!

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] [CODEOWNERS](./CODEOWNERS) updated (for new stuff)
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
Thanks for the PR!

We're looking to make the base `Dockerfile` serve the production build through nginx instead though, with nginx config in the top-level of the repo, and copying in only `packages/app/build`.

Would be happy to merge this and build on top of it, or if you want to update the PR that works too, let us know 😊

**Comment by inductor** [comment]
@Rugvip
Hi,

I can work on that!

**Review by Rugvip — APPROVED** [review]
Looks good, thanks! :grin:

File: README.md [hunk 1]
```diff
@@ -57,6 +57,17 @@ $ yarn start
 
 The final `yarn start` command should open a local instance of Backstage in your browser, otherwise open one of the URLs printed in the terminal.
 
+### (Optional)Try on Docker
+
+Run the following commands if you have Docker environment
+
+```bash
+$ docker build . -t spotify/backstage
+$ docker run --rm -it -p 80:80 backstage
```
Comment (Kantis) [inline README.md]: Should image be `spotify/backstage` on this line as well?

Comment (esttorhe) [reply README.md]: Just adding my 2 cents, but yup. Definitely should be `spotify/basckstage` as it currently won't find the image

**Comment by inductor** [comment]
@Kantis @esttorhe Fixed

**Pull request merged**